ENOENT when using `bun pm pack`

I'm not sure if my configuration is wrong or if this is a bug, but I'm getting this error in CI for my [`react-querybuilder` repo](https://github.com/react-querybuilder/react-querybuilder):

```
ENOENT: no such file or directory, open '/home/runner/work/react-querybuilder/react-querybuilder/packages/antd/@react-querybuilder-antd-8.13.0.tgz'
```

(The `antd` package is the first one alphabetically so I assume all the others would also fail.)

The latest CI run is here (relevant logs below). The build completes successfully but then pkg-pr-new can't find the `tgz` file.

https://github.com/react-querybuilder/react-querybuilder/actions/runs/20473533478/job/58833752532

The GHA config is here:

https://github.com/react-querybuilder/react-querybuilder/blob/1711c8a3e906733106ec565eacf6dfcff4e573a4/.github/workflows/main.yml#L24

Relevant code:

```yml
jobs:
  Build:
    runs-on: ubuntu-latest
    steps:
      - uses: actions/checkout@v6
      - uses: oven-sh/setup-bun@v2
      - run: bun install --frozen-lockfile
      - run: bun run build
      - run: bunx pkg-pr-new publish './packages/*' --template './examples/*' --packageManager=bun --bun
```

Partial GHA log:

```
2025-12-23T23:04:14.9482419Z ##[group]Run bunx pkg-pr-new publish './packages/*' --template './examples/*' --packageManager=bun --bun
2025-12-23T23:04:14.9483116Z [36;1mbunx pkg-pr-new publish './packages/*' --template './examples/*' --packageManager=bun --bun[0m
2025-12-23T23:04:14.9550694Z shell: /usr/bin/bash -e {0}
2025-12-23T23:04:14.9550926Z ##[endgroup]
2025-12-23T23:04:16.1251367Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/antd@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:16.4470599Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/bootstrap@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:16.8311925Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/bulma@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:17.1104226Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/chakra@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:17.4374341Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/core@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:17.8146180Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/datetime@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:18.1152208Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/dnd@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:18.4364469Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/drizzle@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:18.7307091Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/fluent@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:19.0070304Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/mantine@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:19.3069130Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/material@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:19.5955190Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/native@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:19.8809446Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/react-querybuilder@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:20.1670832Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/rules-engine@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:20.4428673Z Unexpected response checking https://pkg.pr.new/react-querybuilder/react-querybuilder/@react-querybuilder/tremor@1711c8a3e906733106ec565eacf6dfcff4e573a4 (404)
2025-12-23T23:04:20.4448622Z preparing template: react-querybuilder-template-example
2025-12-23T23:04:20.4559282Z preparing template: react-querybuilder-antd-example
2025-12-23T23:04:20.4622055Z preparing template: react-querybuilder-base-ui-example
2025-12-23T23:04:20.4686928Z preparing template: react-querybuilder-basic-example
2025-12-23T23:04:20.4734394Z preparing template: react-querybuilder-basic-ts-example
2025-12-23T23:04:20.4786138Z preparing template: react-querybuilder-bootstrap-example
2025-12-23T23:04:20.4833445Z preparing template: react-querybuilder-bulma-example
2025-12-23T23:04:20.4882903Z preparing template: react-querybuilder-chakra-example
2025-12-23T23:04:20.4936795Z preparing template: react-querybuilder-ci
2025-12-23T23:04:20.4995820Z preparing template: react-querybuilder-dnd-example
2025-12-23T23:04:20.5057060Z preparing template: react-querybuilder-fluent-example
2025-12-23T23:04:20.5112939Z preparing template: react-querybuilder-mantine-example
2025-12-23T23:04:20.5164708Z preparing template: react-querybuilder-material-example
2025-12-23T23:04:20.5230520Z preparing template: react-querybuilder-native-example
2025-12-23T23:04:20.5274560Z preparing template: react-querybuilder-next-example
2025-12-23T23:04:20.5327336Z preparing template: react-querybuilder-basic-ts-example
2025-12-23T23:04:20.5370631Z preparing template: react-querybuilder-tremor-example
2025-12-23T23:04:20.6114643Z [error] ENOENT: no such file or directory, open '/home/runner/work/react-querybuilder/react-querybuilder/packages/antd/@react-querybuilder-antd-8.13.0.tgz'
2025-12-23T23:04:20.6115925Z   at async open (node:internal/fs/promises:639:25)
2025-12-23T23:04:20.6116573Z   at async Object.readFile (node:internal/fs/promises:1246:14)
2025-12-23T23:04:20.6117239Z   at async resolveTarball (node_modules/pkg-pr-new/dist/index.js:3709:44)
2025-12-23T23:04:20.6118319Z   at async Object.run (node_modules/pkg-pr-new/dist/index.js:3524:44)
2025-12-23T23:04:20.6118942Z   at async runCommand (node_modules/pkg-pr-new/dist/index.js:2938:16)
2025-12-23T23:04:20.6119558Z   at async runCommand (node_modules/pkg-pr-new/dist/index.js:2929:11)
2025-12-23T23:04:20.6120218Z   at async runMain (node_modules/pkg-pr-new/dist/index.js:3065:7) 
2025-12-23T23:04:20.6120574Z 
2025-12-23T23:04:20.6121038Z [error] ENOENT: no such file or directory, open '/home/runner/work/react-querybuilder/react-querybuilder/packages/antd/@react-querybuilder-antd-8.13.0.tgz'
2025-12-23T23:04:20.6209814Z ##[error]Process completed with exit code 1.
```

--- a/packages/cli/index.ts
+++ b/packages/cli/index.ts
@@ -676,7 +676,7 @@ async function resolveTarball(pm: PackMethod, p: string, pJson: PackageJson) {
   if (pm === "yarn") {
     cmd += ` --filename ${filename}`;
   } else if (pm === "bun") {
-    cmd = `bun pm pack --filename ${filename}`;
+    cmd = `bun pm pack --quiet --filename ${filename}`;
   }
   const { stdout } = await ezSpawn.async(cmd, {
     stdio: "overlapped",
@@ -691,6 +691,10 @@ async function resolveTarball(pm: PackMethod, p: string, pJson: PackageJson) {
     const tgzFiles = fsSync
       .readdirSync(p)
       .filter((file) => file.endsWith(".tgz"));
+    const bunFilename = stdout.trim();
+    if (bunFilename) {
+      filename = bunFilename;
+    }
     console.warn(`[bun pack] stdout:\n${stdout}`);
     console.warn(
       `[bun pack] expected filename: ${filename}; tgz files: ${tgzFiles.join(", ") || "(none)"}`,

